docs/conf.py

```diff
@@ -25,7 +25,7 @@ def __getattr__(cls, name):
 MOCK_MODULES = ['numpy', 'scipy', 'matplotlib', 'matplotlib.pyplot',
                 'sklearn.grid_search', 'sklearn.externals', 'plotly',
                 'plotly.graph_objs', 'matplotlib.gridspec', 'scikit-learn',
-                'pandas']
+                'pandas', 'plotly.plotly', 'plotly.tools', 'Ipython.display']
```

Comment:
Just for future info, it's essential to mock any module you import that may have a C component within conf.py (add them to this list) to get the docs to build, because autodoc